sdk_contrib/inmemory_export/src/main/java/io/opentelemetry/sdk/contrib/trace/export/SpanDataUtils.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.contrib.trace.export;
+
+import static io.opentelemetry.trace.AttributeValue.booleanAttributeValue;
+import static io.opentelemetry.trace.AttributeValue.doubleAttributeValue;
+import static io.opentelemetry.trace.AttributeValue.longAttributeValue;
+import static io.opentelemetry.trace.AttributeValue.stringAttributeValue;
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Timestamp;
+import io.opentelemetry.proto.trace.v1.AttributeValue;
+import io.opentelemetry.proto.trace.v1.Span;
+import io.opentelemetry.proto.trace.v1.Span.Attributes;
+import io.opentelemetry.proto.trace.v1.Span.TimedEvent;
+import io.opentelemetry.proto.trace.v1.Span.TimedEvent.Event;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.trace.Link;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.TraceFlags;
+import io.opentelemetry.trace.TraceId;
+import io.opentelemetry.trace.Tracestate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+// TODO - Properly set TraceFlags/Tracestate for SpanContext and Links.
+final class SpanDataUtils {
+  private SpanDataUtils() {}
+
+  public static SpanData getFromProto(Span span) {
```

Comment:
why the suffix `FromProto` for these methods?  That implies some relationship to protobuf for me...  Perhaps `FromSpan` instead?

Reply:
The Span instance being passed in is a protobuf instance. Maybe name the parameter `protobufSpan` to make it clear?

Reply:
Correct, it's a `io.opentelemetry.proto.trace.v1.Span` instance ;) Yeah, I could actually use `protoSpan` to make things more obvious :)

Reply:
Ah, I missed that part.